open-reac/src/main/java/com/powsybl/openreac/parameters/output/OpenReacResult.java

```diff
@@ -82,7 +83,19 @@ public List<VscConverterStationModification> getVscModifications() {
         return vscModifications;
     }
 
-    public List<NetworkModification> getAllModifs() {
+    public Map<String, Pair<Double, Double>> getVoltagePlan() {
+        return voltagePlan;
+    }
+
+    public boolean getWarmStat() {
```

Comment:
What is the aim of this parameter? the name is confusing, I was thinking about something like `update-network-with-voltages` or what you prefer.